docs/50-advanced/25-compose-to-acorn.md

```diff
@@ -18,6 +18,7 @@ In Docker Compose, a service is a where containers are defined. In Acorn, contai
 
 ### Compose Networks
 
+// TODO - same here. ok to keep?
```

Comment:
We could drop it without causing confusion. We just need to say that it's a single network per project region as far as the user is concerned.